benches/algos.rs

```diff
@@ -3,7 +3,7 @@
 extern crate pathfinding;
 extern crate test;
 
-use pathfinding::prelude::{astar, bfs, dfs, dijkstra, fringe, idastar};
+use pathfinding::prelude::{astar, bfs, dfs, iddfs, dijkstra, fringe, idastar};
```

Comment:
That's not in the right order.

Reply:
Oh! I'm using that ordering since it made sense from a conceptual standpoint. But I see now that you're using an alphabetical ordering? I'll follow suit then :)